### ISSUE TITLE
Add new event for app is closed or moved to the background

### ISSUE THREAD

**Issue by g-opzt** [department-of-veterans-affairs/va-mobile-app#6225]
## ACs
- [ ] Add a new event called `vama_background` to capture when users are moving the app from the background of their 

See thread here for discussion on why we are not currently going to track closures.

**Comment by Sparowhawk** [comment]
@g-opzt @narin 

Looking through this I was able to determine that we can only log when a user backgrounds the app. There is no current way in react native to log when the user or OS kills the app as the native code is killed first before it gets to the react native portion. That being said potential solutions would be to hook up native firebase for Android and iOS respectively which may be fairly difficult or we can be fine with only getting when the user backgrounds the app.

**Comment by g-opzt** [comment]
Yes, we can go ahead with just tracking moving to the background. I will update the description here so the event better reflects what we're tracking.

**Comment by timwright12** [comment]
@OVezeamama Can you review and decide if this is still data we want to track? I'm unclear what we're looking for with this information

**Comment by OVezeamama** [comment]
@timwright12 Yes we will be going ahead with this. Goal is to see analytics for app closure and exits.

**Comment by OVezeamama** [comment]
@theodur Is this something you can assist with?

**Comment by theodur** [comment]
@OVezeamama Yeah, I or any of the other global FE engineers can do this. It'll probably need to be prioritized by @ajsarkar28 for an upcoming sprint, or one of us would potentially get to this is we finish all the sprint work early

**Comment by OVezeamama** [comment]
Will touch base with @ajsarkar28 to discuss priorities. Thanks
